vnext/local-cli/runWindows/utils/msbuildtools.js

```diff
@@ -73,20 +73,40 @@ class MSBuildTools {
       checkRequirements.isWinSdkPresent('10.0');
     } catch (e) {
       newError(e.message);
-      return;
+      throw e;
     }
 
-    console.log(`Running MSBuild with args ${args.join(' ')}`);
+    if (verbose) {
+      console.log(`Running MSBuild with args ${args.join(' ')}`);
+    }
 
     const progressName = 'Building Solution';
     const spinner = newSpinner(progressName);
-    await commandWithProgress(
-      spinner,
-      progressName,
-      path.join(this.path, 'msbuild.exe'),
-      [slnFile].concat(args),
-      verbose,
-    );
+    try {
+      await commandWithProgress(
+        spinner,
+        progressName,
+        path.join(this.path, 'msbuild.exe'),
+        [slnFile].concat(args),
+        verbose,
+      );
+    } catch (e) {
+      let error = e;
+      if (!e) {
+        error = new Error(
+          fs
+            .readFileSync(errorLog)
+            .toString()
+            .split(EOL)[0],
+        );
+        error.logfile = errorLog;
+      }
+      throw error;
+    }
+    // If we have no errors, delete the error log when we're done
+    if (fs.statSync(errorLog).size === 0) {
+      fs.unlinkSync(errorLog);
```

Comment:
If we're already async, could we await fs.promises.unlink instead? That api is stable as of Node 10

Reply:
Same for the above readFileSync.